05/DB.test.js

```diff
@@ -1 +1,155 @@
-import DB from './DB';
\ No newline at end of file
+import DB from "./DB";
+
+describe("DB", () => {
+    it("check if insert row", async () => {
+        const newDB = new DB();
+
+        await newDB.insert({ city: "Warsaw" });
+        const rows = await newDB.getRows();
+        expect(rows.length).toBe(1);
+    });
+    it("if row is duplicated throw an error", async () => {
+        const newDB = new DB();
+
+        const row = { city: "Cracow", id: 2 };
+        const errorMsg = "ID can't be duplicated!";
+
+        try {
+            await newDB.insert(row);
+            await newDB.insert(row);
+        } catch (error) {
+            expect(error).toBe(errorMsg);
+        }
```

Comment:
Ponieważ tutaj jest warunek (jeśli nie będzie błędu to catch się nie wykona) to trzeba użyć `expect.assertions(1);`